### DISCUSSION TITLE
ipv6

### DISCUSSION THREAD

**Discussion by leonidcan** [FlareSolverr/FlareSolverr#199] [Q&A]
Please help setup ipv6 in docker
It seems to work fine from cmd line / sources but when I run in container ip6 
I am getting this error "Unable to process browser request. Error: Error: net::ERR_NAME_NOT_RESOLVED"

**Comment by ngosang** [comment]
Here you have an example => https://github.com/bhcopeland/ansible-seedbox-env/blob/12d771d88f3b54316db1950cee1ec8d58771bcf8/files/docker/docker-flaresolverr/docker-compose.yml#L17
